modules/rhel-compute-updating.adoc

```diff
@@ -2,26 +2,30 @@
 //
 // * updating/updating-cluster-rhel-compute.adoc
 
-[id="rhel-compute-updating_{context}"]
-= Updating RHEL compute machines in your cluster
+[id="rhel-compute-updating-minor_{context}"]
+= Updating {op-system-base} compute machines in your cluster
 
-After you update your cluster, you must update the Red Hat Enterprise Linux (RHEL)
-compute machines in your cluster.
+After you update your cluster, you must update the {op-system-base-full} compute machines in your cluster.
+
+[IMPORTANT]
+====
+Because only {op-system-base-full} version 7.9 or later is supported for worker (compute) machines, you must not upgrade the {op-system-base} worker machines to version 8.
+====
+
+You can also update your compute machines to another minor version of {product-title} if you are using {op-system-base} as the operating system. You do not need to exclude any RPM packages from {op-system-base} when performing a minor version update.
 
 .Prerequisites
 
 * You updated your cluster.
 +
 [IMPORTANT]
 ====
-Because the RHEL machines require assets that are generated by the cluster to
-complete the update process, you must update the cluster before you update
-the RHEL compute machines in it.
+Because the {op-system-base} machines require assets that are generated by the cluster to complete the update process, you must update the cluster before you update the {op-system-base} worker machines in it.
 ====
 
-* You have access to the machine that you used to add the RHEL compute machines
-cluster. You must have access to the `hosts` Ansible inventory file that defines
-your RHEL machines and the `upgrade` playbook.
+* You have access to the machine that you used to add the {op-system-base} compute machines cluster. You must have access to the `hosts` Ansible inventory file that defines your {op-system-base} machines and the `upgrade` playbook.
```

Comment:
I was a little thrown off by the first sentence. I see this wording was there before this PR, but just wanted to flag it. Would something like this make more sense?

`You have access to the local machine that you used to add the {op-system-base} compute machines to your cluster.`

Reply:
Thanks, @codyhoag - I agree that it was an awkward sentence and I like your suggested edit, so I've used your wording. 😁